src/main/java/az/edu/turing/modul1/Task1A.java

```diff
@@ -0,0 +1,58 @@
+package az.edu.turing.modul1;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Task1A {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("-----------LET THE GAME BEGIN---------------\n");
+
+
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);
+        //  int randomNumber = (int) (Math.random() * 100);
+
+
+        System.out.print("Enter the name: ");
+        String name = sc.nextLine();
+
+        System.out.println("Enter the number you want to,between 0 and 100");
+        Integer number = sc.nextInt();
+
+        int[] numbers = new int[100];
+        int count = 0;
+        numbers[count] = number;
+        count++;
```

Comment:
instead you can write `numbers[count++] = number;`
also you write same code in 43,44 lines. Then we should avoid this dublication

Reply:
if I write  as you say, the first element will not be  add to finalArrays

Reply:
you can implement logic for prevent that, pls think about it, also move input validation to another method